server/metrics/metrics.go

```diff
@@ -401,6 +410,19 @@ func (m *Metrics) MetricResourcesStalledAtStage(stage string) {
 	c.Inc()
 }
 
+func (m *Metrics) MetricStorageContainerType(imageStore string) {
+	c, err := m.metricContainerStorageType.GetMetricWithLabelValues("container_storage")
+	if err != nil {
+		logrus.Warnf("Unable to write container storage metric: %v", err)
+	}
+	var typeOfStorage float64 = 0
```

Comment:
no way to set separate filesystem now?

Reply:
There is. Do you know if we expose the constant we use for graphroot anywhere?

Reply:
I didn't add it because we aren't technically supporting this in any feature at the moment.

Reply:
in
```
vendor/github.com/containers/storage/types/options_linux.go:	defaultGraphRoot string = "/var/lib/containers/storage"

```
so not exposed. the other thing to think about is we don't know if it's a split filesystem if we don't check the disk is  separated...

Reply:
Yea but imo from the crio side, I don't think that is that important of an distinction.

We wouldn't behave any differently if /var/lib/images was on same disk or not. I think the logic I have for imageStore is okay.

Reply:
yeah I mostly mean for split fs when graphroot is on a separate disk